en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md

```diff
@@ -918,6 +918,126 @@ To use the Google Ads connector, first create the connection with your configura
     }
     ```
 
+??? note "uploadUserData"
+    The `uploadUserData` operation uploads the given user data.
+    <table>
+        <tr>
+            <th>Parameter Name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+            <tr>
+            <td><code>customerId</code></td>
+            <td>Required. The ID of the customer for which to update the user data.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>operations</code></td>
+            <td>The list of operations to perform on individual ads.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>customerMatchUserListMetadata</code></td>
+            <td>Metadata for data updates to a Customer Match user list.</td>
+            <td>No</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+
+    ```xml
+    <googleAds.uploadUserData configKey="GOOGLE_ADS_CONN">
+        <customerId>{json-eval($.customer_id)}</customerId>
+        <query>{json-eval($.operations)}</query>
+        <query>{json-eval($.customerMatchUserListMetadata)}</query>
+    </googleAds.uploadUserData>
+    ```
+ 
+    **Sample request**
+
+    ```json
+    {
+        "customer_id": "123123123",
+        "operations": [
+            {
+                "create": {
+                    "userIdentifiers": [
+                        {
+                            "hashedEmail": "2c41b9d011bc28e71842637075e2a67cf4e73010172f4a18985494467d73a6d6"
+                        },
+                        {
+                            "hashedEmail": "00b340221ad566a1400936daadce44a7c61b5b04505fc66d3d55d96bde434bc1"
+                        }
+                    ]
+                }
+            }
+        ],
+        "customerMatchUserListMetadata": {
+            "userList": "customers/123123123/userLists/8827689117"
+        }
+    }
+    ```
+
+??? note "userListsMutate"
+    The `userListsMutate` operation creates or updates user lists. Operation statuses are returned.
+    <table>
+        <tr>
+            <th>Parameter Name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td><code>customerId</code></td>
+            <td>Required. The ID of the customer whose user lists are being modified.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>operations</code></td>
+            <td>Required. The list of operations to perform on individual user lists. Type: object (UserListOperation)</td>
```

Comment:
Shall we link the object from the Google Ads API doc here to maintain consistency?

Reply:
ec1816096eb3119e728cf4e4b0db4edde3a08ac2